examples/tensorflow_mnist_eager.py

```diff
@@ -17,8 +17,11 @@
 import tensorflow as tf
 import horovod.tensorflow as hvd
 
-
 def main(_):
+    if not hasattr(tf, 'enable_eager_execution'):
```

Comment:
We don't need this check in the example, just in the tests.  We can skip running example for old TF via if statement in `.travis.yml`

Reply:
Let's get rid of this check.